dev/tests/static/testsuite/Magento/Test/Integrity/Magento/Backend/ControllerAclTest.php

```diff
@@ -232,7 +232,7 @@ private function isItTest($relativeFilePath)
      */
     private function getControllerPath($relativeFilePath)
     {
-        if (preg_match('~(Magento\/.*Controller\/Adminhtml\/.*)\.php~', $relativeFilePath, $matches)) {
+        if (preg_match('~(Magento\/[^\/]+\/Controller\/Adminhtml\/.*)\.php~', $relativeFilePath, $matches)) {
```

Comment:
LGTM, but you need apply changes to `2.2-develop` first.

Reply:
Created PR #16707 fir 2.2-develop, thanks